### ISSUE TITLE
Breakpoints do not work when debugging on Python 3.3.5 

### ISSUE THREAD

**Issue by IgorNovozhilov** [Microsoft/ptvsd#828]
## Environment data

- VS Code version: 
![image](https://user-images.githubusercontent.com/3082812/45803066-3e049d80-bcc1-11e8-985c-90f27fa1e15a.png)

- Extension version (available under the Extensions sidebar): 2018.8.0
- OS and version: win7 pro
- Python version (& distribution if applicable, e.g. Anaconda): 3.3.5 (32-bit)

## Actual behavior

Breakpoints do not work when debugging

## Steps to reproduce:
1. open python file
2. to set a breakpoint
3. run as f5

## Logs

```
Microsoft Windows [Version 6.1.7601]
(c) Корпорация Майкрософт (Microsoft Corp.), 2009. Все права защищены.

D:\Dev>cd d:\Dev && cmd /C "set "PYTHONIOENCODING=UTF-8" && set "PYTHONUNBUFFERED=1" && set "PYTHONPATH=c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd" && C:\Python33\python.exe -m ptvsd --host
localhost --port 50271 d:\Dev\debug.py "
Traceback (most recent call last):
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\wrapper.py", line 1254, in done
    fut.result()
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\futures.py", line 40, in result
    reraise(self._exc_info)
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\reraise3.py", line 8, in reraise
    raise exc_info[1].with_traceback(exc_info[2])
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\futures.py", line 155, in callback
    x = it.send(fut.result())
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\wrapper.py", line 1608, in on_stackTrace
    module = self.modules_mgr.add_or_get_from_path(norm_path)
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\wrapper.py", line 601, in add_or_get_from_path
    'package': value.__package__,
AttributeError: 'module' object has no attribute '__package__'
Traceback (most recent call last):
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\wrapper.py", line 1254, in done
    fut.result()
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\futures.py", line 40, in result
    reraise(self._exc_info)
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\reraise3.py", line 8, in reraise
    raise exc_info[1].with_traceback(exc_info[2])
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\futures.py", line 155, in callback
    x = it.send(fut.result())
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\wrapper.py", line 1608, in on_stackTrace
    module = self.modules_mgr.add_or_get_from_path(norm_path)
  File "c:\Users\user1\.vscode\extensions\ms-python.python-2018.8.0\pythonFiles\experimental\ptvsd\ptvsd\wrapper.py", line 601, in add_or_get_from_path
    'package': value.__package__,
AttributeError: 'module' object has no attribute '__package__'
```

**Comment by IgorNovozhilov** [comment]
https://github.com/Microsoft/vscode-python/issues/2636

**Comment by karthiknadig** [comment]
https://github.com/Microsoft/ptvsd/issues/210 Using this to track compatibility work for 3.1-3.3

**Comment by IgorNovozhilov** [comment]
I don't know if this solves all the compatibility issues for this issue #210.
But with this fix, this problem is solved, and I was able to fully debug my code in Python version 3.3.5

Fix debugging on Python 3.3.5 #873

PS: [PEP 366 -- Main module explicit relative imports / Proposed Change](https://www.python.org/dev/peps/pep-0366/#id11)

When launched from the console, the attributes are equal "None"
![04 10 2018 09-10-13-368](https://user-images.githubusercontent.com/3082812/46456455-9ce50f00-c7b7-11e8-9c86-8113f34c264e.png)

When debugging in 3.6, the attribute is present
![04 10 2018 09-24-33-036](https://user-images.githubusercontent.com/3082812/46456457-9eaed280-c7b7-11e8-9fb4-4872577cc9a2.png)

When debugging in version 3.6, the attribute is missing
![04 10 2018 09-26-28-384](https://user-images.githubusercontent.com/3082812/46456462-a1112c80-c7b7-11e8-9dff-431a346fa394.png)
